src/engine/engine.spec.ts

```diff
@@ -2,7 +2,6 @@ import { logging } from '../__mocks__/utils/dev-kit-logger';
 
```

Comment:
I rather have a specific type than `any` as a general rule when I'm coding In Typescript. I want to know the reasons that lead you to use `any`, maybe I am not seeing something.

Reply:
In test files I often use `any` when I want to create partial options just containing what's needed for the test case, it's easier to test some scenarios with the minimum boilerplate, but if you think it's better with a type I can rewrite it as `const baseSchema: Schema = { ... }`.

Reply:
What do you think?